invoicepilot_app/models.py

```diff
@@ -10,10 +10,12 @@ class PaymentTransaction(models.Model):
     # Add more fields as needed
 
 class InvoiceItem(models.Model):
-    invoice = models.ForeignKey('Invoice', on_delete=models.CASCADE, related_name='items')
+    owner = models.ForeignKey(User, on_delete=models.CASCADE)
     description = models.CharField(max_length=255)
     quantity = models.IntegerField()
     price = models.DecimalField(max_digits=10, decimal_places=2)
+    date_created = models.DateTimeField(auto_now_add=True)
+    invoice_date_generated = models.DateTimeField()
```

Comment:
Your code looks mostly correct, but there seems to be a minor issue with the invoice_date_generated field. The invoice_date_generated field is defined as a DateTimeField, but it doesn't specify the auto_now or auto_now_add options, which might lead to an issue when trying to create instances of the InvoiceItem model. 
Your code looks mostly correct, but there seems to be a minor issue with the invoice_date_generated field. The invoice_date_generated field is defined as a DateTimeField, but it doesn't specify the auto_now or auto_now_add options, which might lead to an issue when trying to create instances of the InvoiceItem model. If you intend for invoice_date_generated to be set automatically when an InvoiceItem is created, you should use auto_now_add=True.

Reply:
If there's anything to add to this, it'll be the auto_now method, since we will be updating the field everytime a user downloads the PDF. In the view, I already set it to timezone.now() which will also do as intended. But for best practice, I'll add the auto_now field as you have suggested.